pkg/ui/workspaces/db-console/src/views/reports/containers/range/rangeTable.tsx

```diff
@@ -827,12 +817,6 @@ export default class RangeTable extends React.Component<RangeTableProps, {}> {
             ? "range-table__cell--warning"
             : "",
         ),
-        truncatedIndex: this.createContent(
-          FixLong(info.state.state.truncated_state.index),
```

Comment:
I'm not familiar with these bits - is the a Go code that corresponds to these structs? I think we logically want to replace `info.state.state.truncated_state` with `info.state.truncated_state` (that corresponds to the type of change that happened in the PR linked from the description), but I'm not sure that new field exists here yet.

Reply:
Probably `info.state.state` corresponds to `kvserverpb.ReplicaState`. Do you know what its parent `info.state` maps to?

Reply:
The [RangeInfo](https://github.com/cockroachdb/cockroach/blob/7f75f3ee76df64652eff124c2c5b41836e1311aa/pkg/kv/kvserver/kvserverpb/state.proto#L165)? So we might want to add a field to it.